tpm2tools/keys.go

```diff
@@ -240,39 +272,45 @@ func (k *Key) Unseal(in *proto.SealedBytes) ([]byte, error) {
 	}
 	defer tpm2.FlushContext(k.rw, sealed)
 
-	if len(sel.PCRs) == 0 {
-		return tpm2.Unseal(k.rw, sealed, "")
+	if cOpt != nil {
+		var ticket tpm2.Ticket
+		_, err = tpmutil.Unpack(in.GetTicket(), &ticket)
+		if err != nil {
+			return nil, fmt.Errorf("Ticket unpack failed: %v", err)
+		}
+		hashCon := sessionHashAlg.New()
```

Comment:
Instead of `hashCon` call this `creationHash`

Reply:
done